openlibrary/catalog/marc/parse.py

```diff
@@ -337,18 +339,18 @@ def read_languages(rec: MarcBase, lang_008: str | None = None) -> list[str]:
     for f in rec.get_fields('041'):
         if f.ind2() == '7':
             code_source = ' '.join(f.get_subfield_values('2'))
-            # TODO: What's the best way to handle these?
-            raise MarcException("Non-MARC language code(s), source = ", code_source)
+            logger.error(f'Unrecognised language source = {code_source}')
```

Comment:
I think this will get you what you want; `logger.error`s should appear in sentry , something like: https://sentry.archive.org/organizations/ia-ux/issues/?project=9&project=7&project=10&query=is%3Aunresolved+Got+non-multiple+of+three&referrer=issue-list&statsPeriod=14d